zarr/hierarchy.py

```diff
@@ -523,6 +523,34 @@ def visititems(self, func):
         base_len = len(self.name)
         return self.visitvalues(lambda o: func(o.name[base_len:].lstrip("/"), o))
 
+    def tree(self):
```

Comment:
Consider adding a ``depth`` argument here which defaults to ``None`` (i.e., render the whole subtree) but can be an integer which would limit the tree depth - useful where you only want to view a few levels down, but whole tree might be unwieldy.